Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -466,7 +470,8 @@ TccModePostMemConfig (
     FspsUpd->FspsConfig.TccCacheCfgSize = TccCacheconfigSize;
     DEBUG ((DEBUG_INFO, "Load Tcc Cache @0x%p, size = 0x%x\n", TccCacheconfigBase, TccCacheconfigSize));
 
-    if (IsPchS ()) {
+    if (IsPchS () ||
+      (IsPchP() && 0 == AsciiStrCmp(GetCpuName(),"RaptorLake"))) {
```

Comment:
Should we totally remove the condition check? 
and it looks TccMode is deprecated from the FSP header file. I guess it is not used by FSP at all.